README.md

```diff
@@ -1,2 +1,20 @@
 # comp110-worksheet-5
 Base repository for COMP110 worksheet 5
+
+A) The algorithm checks the elements in a list against a second list with both lists being the same size. If an element from List [i] in one position doesn't match the element from List [j] in the same position. Then the algorithm will return true, indicating that the Lists are not the same. If the opposite occurs, where every element in the lists are the same. Then the algorithm will return false, indicating that the lists are the same.
```

Comment:
In fact there is only one list -- the algorithm checks it for duplicate elements